docs/changes.rst

```diff
@@ -47,6 +47,14 @@ Released: not yet
 
 * Added a troubleshooting section to the docs.
 
+* Added a hidden '--pdb' general option for having the zhmc command break right
```

Comment:
Isn't that already in the other PR?

Reply:
Yes. Since that other PR was a prereq, I based this PR on the other PR.